webportal/lambda/requirements.txt

```diff
@@ -1,2 +1,2 @@
-chalice==1.0.4
+chalice==1.13.0
```

Comment:
From reading random stack overflow posts, it appears that chalice 1.0.* was tightly tied to python 3.6. 1.13 allows chalice to be used with all versions of python3.